UnitsNet.Tests/CustomCode/LengthTests.FeetInches.cs

```diff
@@ -29,63 +31,93 @@ public void FeetInchesRoundTrip()
+            new object[]{"1′", 1, new CultureInfo("en-US")},                    // Feet only
+            new object[]{"1,000′", 1000, new CultureInfo("en-US")},             // Feet only, with seperator
+            new object[]{"1\"", 0.08333333, new CultureInfo("en-US")},          // Inches only
+            new object[]{"1″", 0.08333333, new CultureInfo("en-US")},           // Inches only
+            new object[]{"0' 1\"", 0.08333333, new CultureInfo("en-US")},       // Inches only
+            new object[]{"0' 1″", 0.08333333, new CultureInfo("en-US")},        // Inches only
+            new object[]{"0′ 1\"", 0.08333333, new CultureInfo("en-US")},       // Inches only
+            new object[]{"0′ 1″", 0.08333333, new CultureInfo("en-US")},        // Inches only
+            new object[]{"1' 1\"", 1.08333333, new CultureInfo("en-US")},       // Normal form
+            new object[]{"1′ 1″", 1.08333333, new CultureInfo("en-US")},        // Normal form
+            new object[]{" 1′ 1″ ", 1.08333333, new CultureInfo("en-US")},      // Normal form, requires trimming
+            new object[]{"1'1\"", 1.08333333, new CultureInfo("en-US")},        // Without space
+            new object[]{"1′1″", 1.08333333, new CultureInfo("en-US")},         // Without space
+            new object[]{"1 ft 1 in", 1.08333333, new CultureInfo("en-US")},
+            new object[]{"1ft 1in", 1.08333333, new CultureInfo("en-US")},
+            new object[]{"-1'", -1, new CultureInfo("en-US")},                  // Feet only
+            new object[]{"-1′", -1, new CultureInfo("en-US")},                  // Feet only
+            new object[]{"-1\"", -0.08333333, new CultureInfo("en-US")},        // Inches only
+            new object[]{"-1″", -0.08333333, new CultureInfo("en-US")},         // Inches only
+            new object[]{"-0' 1\"", -0.08333333, new CultureInfo("en-US")},     // Inches only
+            new object[]{"-0' 1″", -0.08333333, new CultureInfo("en-US")},      // Inches only
+            new object[]{"-0′ 1\"", -0.08333333, new CultureInfo("en-US")},     // Inches only
+            new object[]{"-0′ 1″", -0.08333333, new CultureInfo("en-US")},      // Inches only
+            new object[]{"-1' 1\"", -1.08333333, new CultureInfo("en-US")},     // Normal form
+            new object[]{"-1′ 1″", -1.08333333, new CultureInfo("en-US")},      // Normal form
+            new object[]{" -1′ 1″ ", -1.08333333, new CultureInfo("en-US")},    // Normal form, requires trimming
+            new object[]{"-1'1\"", -1.08333333, new CultureInfo("en-US")},      // Without space
+            new object[]{"-1′1″", -1.08333333, new CultureInfo("en-US")},       // Without space
+            new object[]{"-1 ft 1 in", -1.08333333, new CultureInfo("en-US")},
+            new object[]{"-1ft 1in", -1.08333333, new CultureInfo("en-US")},
+            new object[]{"1’000′", 1000, new CultureInfo("de-CH")},             // Feet only, with seperator
```

Comment:
```csharp
new object[]{"1’000′", 1000, new CultureInfo("de-CH")},             // Feet only, with seperator
new object[]{"1’000′ 6\"", 1000.5, new CultureInfo("de-CH")},       // Normal form, using separators for culture
```
Are the new tests with culture specific separators. The rest are all en-US format so being explicit.

Reply:
@pgrawher also pointed out that in some Windows/Linux/Mac versions, the apostrophe symbol can vary for the same culture. But, unless we actually run into this issue on our own machines and the AppVeyor VMs, then I think a hard coded string is better. I really don't expect this character to change much over time for a given culture unless they are fixing a consistency bug in one of the OS versions, but I am merely speculating.

Reply:
For the sake of reviewing this PR, it would be better if you did the refactoring to memberdata in a separate PR, but it helps to know that these were the only two new cases plus the new culture parameter.

Reply:
Aren't you missing the problematic test case? Where the thousand separator character exactly matches the foot or inches unit? Requires a culture that has this exact character though, can construct our own if necessary.

```
new object[]{"1′000′", 1000, new CultureInfo("de-CH")},
```

Reply:
Use `new CultureInfo("...", false)` everywhere. This makes sure that at least when developers use a similarly current version of Windows, the settings will be the same. (This ignores any manual overrides in control panel)